Client/TeamTalkAndroid/src/main/java/dk/bearware/backend/TeamTalkService.java

```diff
@@ -936,21 +936,23 @@ public void onCmdUserLoggedIn(User user) {
                 activecmds.put(cmdid, CmdComplete.CMD_COMPLETE_UNSUBSCRIBE);
         }
 
+        String name = Utils.getDisplayName(getBaseContext(), user);
```

Comment:
If you have a server with 50+ users then this will generate a lot of entries in the chat-log. Are you sure you want this?